lib/rules/max-empty-lines/index.js

```diff
@@ -161,7 +161,7 @@ const rule = (primary, secondaryOptions, context) => {
 			const emptyLFLines = '\n'.repeat(repeatTimes);
 			const emptyCRLFLines = '\r\n'.repeat(repeatTimes);
 
-			return /(\r\n)+/g.test(str)
+			return /(?:\r\n)+/.test(str)
```

Comment:
See [regexp/no-useless-flag](https://ota-meshi.github.io/eslint-plugin-regexp/rules/no-useless-flag.html) - `g` flag is useless for `.test()`